Source/Matter/EMTensor.hpp

```diff
@@ -7,34 +7,64 @@
 #define EMTENSOR_HPP
 
 #include "CCZ4Geometry.hpp"
+#include "CCZ4RHSWithMatter.hpp"
 #include "Cell.hpp"
 #include "FourthOrderDerivatives.hpp"
-#include "Interval.H"
-#include "MatterCCZ4.hpp"
+#include "Interval.hpp"
+#include "VarsTools.hpp"
 #include "simd.hpp"
 
+// AMReX Includes
+#include <AMReX_MultiFab.H>
+
 //! Calculates the EM tensor and then saves the ones specified in the
 //! constructor on the grid
 template <class matter_t> class EMTensor
 {
   public:
     template <class data_t>
-    using Vars = typename MatterCCZ4<matter_t>::template Vars<data_t>;
+    using Vars = typename CCZ4RHSWithMatter<matter_t>::template Vars<data_t>;
+
+    /// derive record name
+    static inline const std::string name = "EMTensor";
+
+    /// Variable names
+    static inline amrex::Vector<std::string> var_names = {"rho"};
+
+    /// 3 components for the momentum density: Si_11, Si_22, Si_33
+    /// 6 components for the stress energy tensor (symmetric): Sij_11, Sij_12
+    /// etc.
+    static inline const amrex::Vector<std::string> extra_var_names = {
+        "Sij_11, Sij_22, Sij_33, Sij_11, Sij_12, Sij_13, Sij_22, Sij_23, "
+        "Sij_33"};
```

Comment:
I think we've agreed on the following names.

```suggestion
    static inline const amrex::Vector<std::string> extra_var_names = {
        "j_1, j_2, j_3, S_11, S_12, S_13, S_22, S_23, S_33"};
```

Reply:
Based on this comment: 
> We should make sure our var names are consistent with the [`emtensor_t` struct](https://github.com/GRTLCollaboration/GRTeclyn/blob/8c6aaba60bb5951be3b1247bda474a2dba1c5978/Source/CCZ4/CCZ4Geometry.hpp#L15-L21).

I thought we had decided to make them consistent with the existing names without matter? If you prefer the j's, I can open a pull request to change them both?

Reply:
Let's just change it to my suggestion in the `emtensor_t` struct in this PR.

Reply:
Could you change the `emtensor_t` struct in CCZ4Geometry.hpp to
```cpp
template <class data_t> struct emtensor_t
{
    Tensor<2, data_t> S;  //!< S_ij = T_ij
    Tensor<1, data_t> j;  //!< j_i = T_ia_n^a
    data_t trS;           //!< trS = S^i_i
    data_t rho;           //!< rho = T_ab n^a n^b
};
```
and modify where this struct appears accordingly?

Reply:
I opened a new PR for this (#117) because `CCZ4Geometry.hpp` wasn't part of this original PR.